openmls/src/group/mls_group/mod.rs

```diff
@@ -229,31 +304,491 @@ impl MlsGroup {
         if !self.is_active() {
             return Err(MlsGroupStateError::UseAfterEviction);
         }
-        self.group
-            .public_group()
+        self.public_group
             .leaf(self.own_leaf_index())
             .map(|node| node.credential())
             .ok_or_else(|| LibraryError::custom("Own leaf node missing").into())
     }
 
+    /// Get the required capabilities extension of this group.
+    pub(crate) fn required_capabilities(&self) -> Option<&RequiredCapabilitiesExtension> {
```

Comment:
`pub(crate)` methods should not be among the public methods. This also applies to quite a few other methods below.

Reply:
Good point. I've organized the individual functions into its own impl block.